### DISCUSSION TITLE
FN and M hotkeys stopped working

### DISCUSSION THREAD

**Discussion by sunriderrr** [seerge/g-helper#740] [Q&A]
I've got Asus G16 2023. Via Armory Crate app I bound hotkeys M3 and M4 for screen brightness adjustment. And M5 was launching Armory Crate by default. Then I tried to use G-Helper but decided to stay with Armory Crate. 
But now M3, M4, and M5 buttons don't respond at all. I've tried to rebind them again via Armory Crate but it didn't help. Moreover, some of the FN keys stopped working either! E.g. now I can't adjust the keyboard brightness with FN+F2/F3 (but it's still changeable via the G-Helper toggle) and I can't change the screen brightness with FN+F7/F8.
The laptop is rebooted, G-Helper is not launched at the moment.
I need help. How to restore the default functionality of all the keys?

UPD:
My apologies! Finally, I remembered that there was an option: "Extra" - "Asus Services" - "Stop" which I actually had used. I got back, pressed "Start" and all the basic functions were restored. Phew! (I didn't expect that this option would keep acting even after rebooting.)

**Comment by seerge** [comment]
@sunriderrr hello, even when you stop asus services app should try to take over hotkeys actually :) 

**P.S.** Sorry, I have read only first part of your comment. If you want to go with AC, you indeed need all asus services running, but mainly Asus Optimization Service (as it handles hotkeys). So you indeed can start them either from g-helper itself (under Extra) or by running https://github.com/seerge/g-helper/blob/main/bloat.bat manually (which does same thing but as a bat file) :)

**Comment by Dreugui** [comment]
THANK YOU ! So much ! I had the same problem. tried to reinstal AC, allmost reset My PC. 
I just reDDL G-helper, stoped and relauch the AC in the option and it fixed everything !
